paper/paper.md

```diff
@@ -0,0 +1,109 @@
+---
+title: 'MNE-BIDS'
+tags:
+  - Python
+  - BIDS
+  - MNE
+  - neuroimaging
+  - MEG
+  - EEG
+authors:
```

Comment:
missing many authors. I'm assuming it's still WIP

Reply:
maybe tag the authors so that they can give you the affiliation in "suggesting" mode

Reply:
> missing many authors. I'm assuming it's still WIP

indeed, see the "What needs to be done" heading in my OP

But good call, I'll tag people.

Reply:
done, see my updated OP